ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js

```diff
@@ -31,6 +31,15 @@ export default {
     },
   ],
   triggers: [
+    {
+      id: 'DelubrumSav Seeker Verdant Tempest',
```

Comment:
Usually I like to put aoes on castbars becuase it's simpler, but I think it might be worth putting this on a 10 second beforeSeconds timeline trigger instead, so that folks have time to heal up and throw mitigation?  What do you think?

Similarly the other aoes below?

Reply:
Trinity Avowed (TA) and Queen fight may need these. The other fights the AoE does considerably less damage, more so when everyone is using Pure Essences. In TA and Queen there is an AoE that also applies a bleed and the damage needs considerably more mitigation.

Reply:
I have changed the AoE + Bleeds spell triggers to match e11s.

Reply:
That's fair, but I think even if they say bigAoe, the TA and Queen ones should be timeline triggers as I mentioned above to give people more time (although arguably all aoes in the instance could be timeline triggers for consistency, imo.)

Reply:
The shortest AoE cast is Malediction Of Agony at 4.7s. The others are at least 5 seconds, with the majority being 5.5 seconds. Gods Save The Queen being the longest at 6 seconds. Use of a timelineTrigger seems unnecessary given that some examples of timelineTriggers in cactbot I found in ultimate are using 5 seconds before.

The biggest issue I think will be The Queen's ending AoEs. Could look to how other raid end tier bosses are handled, but then again these are in quick succession and we have more mitigation from lost actions available to us here. Calling out all the AoEs would lead to a callout every 3 seconds, which I think is a bit excessive compared to previous encounters. The timeline I think could use some cleaning up here as well. The AoEs themselves can be grouped into three sets:
Set 1: Double AoE + Double AoE
-- 5 second "calm" --, there is no real break the next cast starts immediately at end of set
Set 2: Double Aoe + Double AoE + Big AoE (AoE + Bleed)
-- 6.3 second "calm" --, technically 1.3 seconds of no casts
Set 3: Four single AoEs during bleed

So at the minimum I think 3 callouts would be alright for the ending AoEs, which would need to be done by a timelineTrigger.

The Guard's AoEs appear to do the same damage as in Queen's Guard encounter.

Reply:
> The shortest AoE cast is Malediction Of Agony at 4.7s. The others are at least 5 seconds, with the majority being 5.5 seconds. Gods Save The Queen being the longest at 6 seconds. Use of a timelineTrigger seems unnecessary given that some examples of timelineTriggers in cactbot I found in ultimate are using 5 seconds before.

I think the difference here is more that raises are really hard to come by here and the instance is really long, so penalties for messing up are just really high.  I've had a bunch of people ask me to give more warning before raidwides (since if folks are really low and you need two emergency gcd heals, 5s might not be enough).  So, I thought maybe it'd be worth doing in this instance especially, more than any other place.

Yeah, it's different, but I kinda think it's a little bit more warranted just because keeping people alive is a lot more important here.

Reply:
I will add a note to this that due to the nature of the current Trinity Avowed timeline with how it jumps to different phases as different times, Glory of Bozja can be double called out or even missed.

Reply:
I'll fix this up.  Double callouts are an unfixed bug with timeline triggers that I forgot.  The issue is that if the timeline syncs you backwards 0.1s in time or something, it doesn't realize it's already called the timeline callout for that time already.  It's not because the Trinity Avowed timeline jumps around, it's just a timeline trigger issue.

Glory of Bozja should never be missed.  I could imagine if it was at the beginning of the block and you jumped so that it was happening sooner than 7 seconds, then it could get missed.  However, it always comes at the end of a block, and so 7 seconds before it will never be missed.  If this is happening, I think I need more information on why.

Would you mind filing new issues for these when you find bugs.  I can respond here, but it's easy for me to lose track of replies on old merged issues.